Keas.Mvc/ClientApp/components/Access/AccessTable.tsx

```diff
@@ -18,6 +17,72 @@ interface IProps {
 
 export default class AccessTable extends React.Component<IProps, {}> {
   public render() {
+    const columns = [
+      {
+        Cell: data => (
+          <Button
+            color='link'
+            onClick={() => this.props.showDetails(data.row.original)}
+          >
+            Details
+          </Button>
+        ),
+        Header: 'Access',
+        filterable: false,
+        maxWidth: 150,
+        resizable: false,
+        sortable: false
```

Comment:
I don't think`sortable` is a valid column option.  Neither for that matter is `filterMethod` or `sortMethod` or several of the other ones.  If you use typescript typings you'll be able to see pretty easily, also the api def is here https://react-table.tanstack.com/docs/api/useFilters#column-options but a little confusing because the sort, filter, and general options are in different places

Reply:
And by use typescript I mean doing `const columns: Column<IAccess>[]` kind of thing.  That'll make sure you aren't using properties that aren't doing anything